ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-stable-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -42,12 +42,10 @@ build_root:
     tag: golang-1.18
 releases:
   latest:
-    prerelease:
+    release:
       architecture: multi
-      product: ocp
-      version_bounds:
-        lower: 4.15.0-0
-        upper: 4.16.0-0
+      channel: candidate
```

Comment:
how about using `fast` channel as start version in upgrade jobs? `fast` channel is the supported channel officially while `candidate` is not.

Reply:
Done, also update other ci yaml files together.